### PR TITLE
fix: migrate new user project access to mantine modal

### PR DESCRIPTION
**Pull request by iampranavdhar**
Closes: #5404 

### Description:
This PR converts a separate new user page to mantine modal.

### Video

https://github.com/lightdash/lightdash/assets/73348574/c0c86815-9646-4147-bb9a-e32432aad352

### CODE REVIEW

**Comment by IrakliJani** [comment]
thanks @iampranavdhar, testing this now!

File: packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx [hunk 1]
```diff
@@ -122,122 +101,98 @@ const ProjectAccessCreation: FC<ProjectAccessCreationProps> = ({
...
+                        Provide access to your project to a new user
+                    </Title>
+                </Group>
+            }
+            size="lg"
+        >
+            <TrackPage
+                name={PageName.INVITE_MANAGEMENT_SETTINGS}
+                type={PageType.MODAL}
+                category={CategoryName.SETTINGS}
```
[the remark this answers is not in the document]
Comment (iampranavdhar) [reply packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx:134]: Didn't get you.

File: packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx [hunk 2]
```diff
@@ -86,32 +67,30 @@ const ProjectAccessCreation: FC<ProjectAccessCreationProps> = ({
...
-            methods.reset({ ...methods.getValues() }, { keepValues: true });
+            return;
         }
         if (isSuccess) {
-            setEmailSelected('');
-            methods.setValue('role', ProjectMemberRole.VIEWER);
+            form.reset();
         }
-    }, [isError, methods, isSuccess, showToastSuccess, setEmailSelected]);
+    }, [isError, isSuccess, form]);
```
[the remark this answers is not in the document]
Comment (iampranavdhar) [reply packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx:75]: Isn't it working?

[the remark this answers is not in the document]
Comment (IrakliJani) [reply packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx:75]: hi @iampranavdhar it works, there's just a lot of changes in the code apart from UI and it requires more testing

File: packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (IrakliJani) [reply packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx:134]: TrackPage and analytics event seems to be new and did not exist before in the file. I think it was copied from user management page

**Comment by IrakliJani** [comment]
Hey @iampranavdhar, thanks for your contribution! At the moment, our team's capacity is limited, so we can't review this PR. We'll consider it in the future. For now, we'll close it, but please feel free to reopen when needed.

File: packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (iampranavdhar) [reply packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx:134]: Yeah true. Have to change it to a different page name related to the project access page :)

**Comment by iampranavdhar** [comment]
Okay, @IrakliJani  Is there any other issue that I can work on but doesn't, need your effort more?

**Pull request closed without merging**